scripts/feature_toggle_report.py

```diff
@@ -43,12 +40,16 @@
     default=None,
     help='specify toggle types if you only want data on certain toggle type',
     )
+@click.option(
+    '--verbose_report', is_flag=True,
+    help="To get more verbosed version of report. This will result in one row per state data(from each new env) for a toggle",
```

Comment:
```suggestion
    help="To get a more verbose version of the report. This will result in one row per state data (from each new env) for a toggle",
```